redis.conf

```diff
@@ -106,8 +106,8 @@ tcp-backlog 511
 # incoming connections. There is no default, so Redis will not listen
 # on a unix socket when not specified.
 #
-# unixsocket /tmp/redis.sock
-# unixsocketperm 700
+ unixsocket /var/run/redis/redis.sock
+ unixsocketperm 700
```

Comment:
Please leave this uncommented, since this is the default behavior.

Reply:
tanks @madolson i'll change it